NineChronicles.Headless.Executable/Store/StoreTypeExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Libplanet.RocksDBStore;
+using Libplanet.Store;
+
+namespace NineChronicles.Headless.Executable.Store
+{
+    public static class StoreTypeExtensions
+    {
+        public static Func<string, IStore> ToStoreConstructor(this StoreType storeType) => storeType switch
```

Comment:
IMO, creating a factory method like `.CreateStore(string path)` instead of returning an anonymous function is more natural for me.

Reply:
I applied it in 8a255aa commit. Could you review again?